docs/configuration.asciidoc

```diff
@@ -680,253 +547,7 @@ can be used for sharing the topology. If you have both Redis and Elasticsearch
-    index: "packetbeat"
-
-    # Optional Redis database number where the events are stored
-    # The default is 0.
-    db: 0
-
-    # Optional Redis database number where the topology is stored
-    # The default is 1. It must have a different value than db.
-    db_topology: 1
-
-    # Optional password to authenticate with. By default, no
-    # password is set.
-    # password: ""
-
-    # Optional Redis initial connection timeout in seconds.
-    # The default is 5 seconds.
-    timeout: 5
-
-    # Optional interval for reconnecting to failed Redis connections.
-    # The default is 1 second.
-    reconnect_interval: 1
-------------------------------------------------------------------------------
-
-
-===== enabled
-
-Boolean option that enables Redis as output. The default is false.
-
-===== host
-
-Host of the Redis server.
-
-===== port
-
-Port of the Redis server.
-
-===== db
-
-Redis database number where the events are published. The default is 0.
-
-===== db_topology
-
-Redis database number where the topology information is stored. The default is 1.
-
-===== index
-
-Name of the Redis list where the events are published. The default is
-`packetbeat`.
-
-===== password
-
-Password to authenticate with. The default is no authentication.
-
-===== timeout
-
-Redis initial connection timeout in seconds. The default is 5 seconds.
-
-===== reconnect_interval
-
-Interval for reconnecting failed Redis connections. The default is 1 second.
-
-==== File Output
-
-[source,yaml]
-------------------------------------------------------------------------------
-output:
-
-  # File as output
-  # Options:
-  # path: where to save the files
-  # filename: name of the files
-  # rotate_every_kb: maximum size of the files in path
-  # number of files: maximum number of files in path
-  file:
-    enabled: true
-    path: "/tmp/packetbeat"
-    filename: packetbeat
-    rotate_every_kb: 1000
-    number_of_files: 7
-------------------------------------------------------------------------------
-
-Dumps the transactions in a file where each transaction is in a JSON format.
-Currently, this output is used for testing, but it can be used as input for
-Logstash.
-
-===== enabled
-
-Boolean option that enables File as output. The default is false.
-
-===== path
-
-Path to the directory where to save the generated files. The option is
-mandatory.
-
-===== filename
-
-Name of the generated files. The default is `packetbeat` and it generates files: `packetbeat`, `packetbeat.1`, `packetbeat.2`, etc.
-
-===== rotate_every_kb
-
-Maximum size in kilobytes of each file. When this size is reached, the files are
-rotated. The default value is 10 MB.
-
-===== number_of_files
-
-Maximum number of files under path. When this number of files is reached, the
-oldest file is deleted and the rest are shifted from last to first. The default
-is 7 files.
+More details about the Beats outputs are available TODO[here].
```

Comment:
Shouldn't we at least have an Outputs header instead of just a single line that most people will miss?